main/unbound/unbound.initd

```diff
@@ -10,10 +7,13 @@ description="unbound is a Domain Name Server (DNS) that is used to resolve host
 description_configtest="Run syntax tests for configuration files only."
 description_reload="Kills all children and reloads the configuration."
 
-
 UNBOUND_BINARY=${UNBOUND_BINARY:-/usr/sbin/unbound}
 UNBOUND_CHECKCONF=${UNBOUND_CHECKCONF:-/usr/sbin/unbound-checkconf}
-UNBOUND_CONFFILE=${UNBOUND_CONFFILE:-/etc/unbound/${SVCNAME}.conf}
+UNBOUND_CONFFILE=${UNBOUND_CONFFILE:-/etc/unbound/${RC_SVCNAME}.conf}
+
+command=${UNBOUND_BINARY}
+command_args=${UNBOUND_OPTS}
+command_background=yes
```

Comment:
It's required when `command_background=yes`. It will start fine this way yes, but OpenRC will not (or diminishingly so) be able to detect when it has crashed. We prefer it this way (`command_background=yes` and forcing daemons on foreground) because it relies less on start-stop-daemon semantics and gives us flexibility to switch globally between supervisor backends in the future.